scripts/jenkins/ses/ansible/roles/ses/tasks/deepsea_setup.yml

```diff
@@ -40,13 +43,19 @@
     group: salt
     mode: 0644
 
+- name: Ensure deepsea minions are reacheable
+  command: "salt '{{ ses_deepsea_minions }}' test.ping --out=text"
+  register: deepsea_minions
+  changed_when: false
+  failed_when: "ansible_hostname ~ ': True' not in deepsea_minions.stdout"
```

Comment:
Does this (`ansible_hostname`) get evaluated?

Reply:
yes, it does. On when conditionals `{{ }}` is not required.

Reply:
yes failed_when is like a condition line, therefore it's already interpreted indeed.

alternatively, one can write

``
deepsea_minions.stdout.find(ansible_hostname ~': True') == -1
``

in this case one could also:

``
deepsea_minions.stdout | select("match", ansible_hostname ~ ".*True")
``

a land of opportunities!